tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingTestsAgainstCentral.java

```diff
@@ -40,16 +41,21 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 import java.util.Comparator;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import static org.awaitility.Awaitility.given;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
 /**
- * Testing pushing a package to central.
+ * Testing pushing, pulling, searching a package from central and installing package to home repository.
  */
-public class PackagingPushTestCase extends IntegrationTestCase {
+public class PackagingTestsAgainstCentral extends IntegrationTestCase {
```

Comment:
Shall we change the file name to something better ?

Reply:
Fixed with 7adce23